uiowa/uiowa
# Feature: z-index fix for site name and logo

Resolves https://github.com/uiowa/uiowa/issues/3446.


# How to test

- `yarn workspace uids_base build`.
- Testing on site using the toggle menu and inline header like grad.uiowa.edu, zoom out your browser to 50% and verify that you an still click on the site name and logo.
